src/test_utils.rs

```diff
@@ -43,6 +43,15 @@ impl Translator<String, bitcoin::PublicKey, ()> for StrKeyTranslator {
         .unwrap();
         Ok(hash)
     }
+
+    fn hash256(&mut self, _hash256: &String) -> Result<hash256::Hash, ()> {
+        // hard coded value
```

Comment:
This comment adds no value, we can see its a hard coded value. Perhaps comment _why_ this value is used or don't comment at all.

Reply:
Removing the comment